mage_ai/server/api/orchestration.py

```diff
@@ -170,7 +170,7 @@ def process_pipeline_runs(
         if additional_results[0].id != results[0].id:
             filter_dates.append(collection[0]['execution_date'])
 
-        last_execution_date = results[-1].execution_date
+        last_execution_date = results[results.count() - 1].execution_date
```

Comment:
what's the difference here?

Reply:
@wangxiaoyou1993 it was erroring for some reason. possibly due to the sqlalchemy upgrade? but i just changed it in case we upgraded